sfa_dash/blueprints/form.py

```diff
@@ -280,6 +282,78 @@ def post(self, uuid):
                                     uuid=uuid))
 
 
+class DownloadForm(BaseView):
+    def __init__(self, data_type):
+        self.data_type = data_type
+        if data_type == 'observation':
+            self.template = 'forms/obs_data_download_form.html'
+            self.metadata_template = 'data/metadata/observation_metadata.html'
+            self.api_handle = observations
+        if data_type == 'forecast':
+            self.template = 'forms/fx_data_download_form.html'
+            self.metadata_template = 'data/metadata/forecast_metadata.html'
+            self.api_handle = forecasts
```

Comment:
these are mutually exclusive and should probably error if data_type is anything else.